bepasty/views/upload.py

```diff
@@ -72,7 +72,7 @@ def post(self):
         data_type = data['type']
 
         # set max lifetime
-        maxlife_value = int(data['maxlife_value'])
+        maxlife_value = int(data.get('maxlife_value', 1))
```

Comment:
sorry, i somehow missed your PR and just found it by accident now.

the question here is: why is maxlife_value missing from post data?

maxlife_unit (see below) is access in the same way and if it does not crash there also, it seems to be always there.

Reply:
also, your fix does not address "empty string", but rather a missing value (so it would be None, except when fixed like in your pull request).